azure-go-webserver-component/Pulumi.yaml

```diff
@@ -0,0 +1,13 @@
+name: ws-go-azure-comp
+runtime: go
+description: Basic example of an Azure web server accessible over HTTP
+template:
+  config:
+    azure:location:
+      description: The Azure location to use
+      default: westus
+    username:
+      description: The username used to configure the Virtual Machine
+    password:
+      description: The password used to configure the Virtual Machine
+      secret: true
```

Comment:
Do you think we should remove this until we have secret support finished?

Reply:
No, these should work as advertised: we allowed secret (i.e. encrypted) config before we had any SDK support for secret values.